target/linux/realtek/dts-5.10/rtl8198c.dtsi

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/interrupt-controller/mips-gic.h>
```

Comment:
interesting, our rtl83xx don't support the ips-gic; and this older one does? curious indeed.